### PR TITLE
[12.0] [ADD] agreement_operating_unit

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
This module adds operating unit information to agreements and service profiles.

Issue #217

### CODE REVIEW

**Review by max3903 — APPROVED** [review]
@nikul-serpentcs Can you remove the empty line at the end of the rst files?

Otherwise :+1:

File: agreement_operating_unit/views/agreement.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <!-- Copyright (C) 2019 Open Source Integrators
+    Copyright (C) 2019 Serpent Consulting Services
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+    <record id="view_agreement_form_inherit" model="ir.ui.view">
+        <field name="name">agreement.form</field>
+        <field name="model">agreement</field>
+        <field name="inherit_id" ref="agreement.agreement_form"/>
+        <field name="arch" type="xml">
+            <field name="partner_id" position="after">
```
Comment (max3903) [inline agreement_operating_unit/views/agreement.xml]:
```suggestion
            <field name="company_id" position="after">
```

Comment (nikul-serpentcs) [reply agreement_operating_unit/views/agreement.xml]: Done

**Review by osi-scampbell — APPROVED** [review]

**Pull request merged**